src/System.Device.Gpio/Directory.Build.targets

```diff
@@ -1,60 +1,18 @@
 <Project>
-  <!-- Targets for building the notsupported assembly -->
-  <Target Name="SetupEnvironmentForNotSupportedAssembly">
```

Comment:
There is one more problem with doing this which we haven't discussed, and that is what will happen when somebody tries to run this on platforms like OSX. I think that we should perhaps still throw not supported there, but I'm not sure what is the best way to do that.

Reply:
It will be hard to test things on OSX, since I don't have a Mac. But I believe if we really do all the testing for a specific hardware at runtime, this should work on OSX the same. And things like Arduino or FT4222 should work just fine. It would just throw an Exception if trying to instantiate i.e. the Windows10Driver, but that's very much the same as if you try this on linux.

Reply:
There's a difference between "the library is not supported on Platform X" vs "there's no hardware available on Platform X for which we have a driver".